lib/Parser.php

```diff
@@ -272,13 +268,13 @@ public function includeFiles(string $document) : string
             function ($match) {
                 $path = $this->environment->absoluteRelativePath($match[1]);
 
+                if (!is_readable($path)) {
+                    throw new Exception(sprintf('Include path "%s" does not exist or is not readable.', $match[1]));
+                }
+
                 if ($this->includeFileAllowed($path)) {
                     $contents = file_get_contents($path);
 
-                    if ($contents === false) {
-                        throw new Exception(sprintf('Could not load file from path %s', $path));
-                    }
```

Comment:
Because we're checking `is_readable()`, I'm not sure this is needed. But, it can be reverted if I'm wrong - not an important detail - trying to remove unnecessary stuff.